### PR TITLE
feat: Helm Chart implemantation for kubernetes

### PR DESCRIPTION
**Pull request by iammuho**
Fixes : https://github.com/mergeability/mergeable/issues/435

**Problem:**

There is no easy way to install Mergeable into the Kubernetes cluster.

**Solution:**

This PR implements a helm-chart to install mergeable into the Kubernetes cluster.

### CODE REVIEW

**Comment by shine2lay** [comment]
@jusx can you please review this? I am out of my depth here.

File: helm/mergeable/values.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+replicaCount: 1
+
+namespace:
+  enabled: true
+  annotations: {}
+  labels: {}
+
+image:
+  repository: "DOCKER_REPO/mergeable"
```
Comment (jusx) [inline helm/mergeable/values.yaml:13]: Is DOCKER_REPO suppose to be an env variable?

Comment (iammuho) [reply helm/mergeable/values.yaml:13]: it's supposed to be filled, no need to get from environment values I guess, but just static typing

**Review by jusx — APPROVED** [review]
LGTM but I have not tested it and I assume you have.

**Pull request merged**